### PR TITLE
New: aria label added for partly correct answers (fixes #553)

### LINKED ISSUE #553: Create aria label for partly correct answers
### Subject of the enhancement
As per https://github.com/adaptlearning/adapt_framework/issues/3565, there is no aria label for partially correct question answers.

### Expected behaviour
An aria label should exist for correct, partially correct and incorrect.

### PR DESCRIPTION
**Pull request by kirsty-hames**
Fixes https://github.com/adaptlearning/adapt-contrib-core/issues/553

### New
Requires `answeredPartlyCorrect` aria label added to course __globals_  (FW PR https://github.com/adaptlearning/adapt_framework/pull/3570).

### Testing
You can test the following either by running a screen reader or inspect the button marking label `.js-btn-marking-label` (button marking displays bewteen the _show answer_ / _feedback_ buttons.

1. Test with FW branch [issue/3565](https://github.com/adaptlearning/adapt_framework/tree/issue/3565)
2. Using the Adapt course, go to Matching component c-225.
3. Answer the Matching component as per below to test correct, partially correct and incorrect aria labels are used. 
  - correct answers - item 1: 2013, item 2: 2008, item 3: Hebrew
  - partly correct answers - item 1: 2013, item 2: 2013, item 3: Esperanto
  - incorrect answer - item 1: 2008, item 2: 2013, item 3: Esperanto

### CODE REVIEW

File: js/views/buttonsView.js [hunk 1]
```diff
@@ -185,13 +186,19 @@ export default class ButtonsView extends Backbone.View {
 
     const $ariaLabel = this.$('.js-btn-marking-label');
     const hasSpanAriaLabel = Boolean($ariaLabel.length);
+    const correctnessAriaLabel = isCorrect
+      ? ariaLabels.answeredCorrectly
+      : isPartlyCorrect
+        ? ariaLabels.answeredPartlyCorrect
```
Comment (oliverfoster) [inline js/views/buttonsView.js:192]:
```suggestion
        ? (ariaLabels.answeredPartlyCorrect ?? ariaLabels.answeredIncorrectly)
```
Just for backward compatibility?

If `ariaLabels.answeredPartlyCorrect` is `null` or `undefined` then use `ariaLabels.answeredIncorrectly`.

**Review by oliverfoster — CHANGES_REQUESTED** [review]

File: js/views/buttonsView.js (hunk 1 above)
Comment (kirsty-hames) [reply js/views/buttonsView.js]: Thanks @oliverfoster. Amended in [ec043c4](https://github.com/adaptlearning/adapt-contrib-core/pull/554/commits/ec043c48a965d48b1de342fd1ebc1b557d158e08).

**Review by oliverfoster — APPROVED** [review]

**Review by cahirodoherty-learningpool — APPROVED** [review]
👀

**Review by chris-steele — APPROVED** [review]

**Pull request merged**